public/js/chess.js

```diff
@@ -17,6 +17,13 @@ const checkMove = ( event ) => {
   }
 }
 
+const reset = () => {
+  manager = new GameManager()
+  renderBoard( manager.board )
+}
+
+
+//Rendering Functions
```

Comment:
Yeah, I was wondering what the best way to encapsulate those functions should be, but didn't have time to go back and refactor them.

The specs asked game logic and rendering/html-focused logic to be separated, so I wasn't sure how to divide up some of the front-end concerns.  I suppose I have some front-end logic that interacts more with the board and some that interacts more with the game manager.